$MirOS: ports/misc/mc/patches/patch-doc_mcedit_1_in,v 1.1.7.1 2005/03/18 15:50:27 tg Exp $
--- doc/mcedit.1.in.orig	Wed Jun  8 12:26:45 2005
+++ doc/mcedit.1.in	Mon Feb 20 18:32:46 2006
@@ -73,13 +73,13 @@ In addition to that, Shift combined with
 (if supported by the terminal):
 .B Ctrl-Ins
 copies to the file
-.BR ~/.mc/cedit/cooledit.clip ,
+.BR ~/.etc/mc/cedit/cooledit.clip ,
 .B Shift-Ins
 pastes from
-.BR ~/.mc/cedit/cooledit.clip ,
+.BR ~/.etc/mc/cedit/cooledit.clip ,
 .B Shift-Del
 cuts to
-.BR ~/.mc/cedit/cooledit.clip ,
+.BR ~/.etc/mc/cedit/cooledit.clip ,
 and
 .B Ctrl-Del
 deletes highlighted text.  Mouse highlighting also works on some
@@ -105,7 +105,7 @@ by pressing that key.  The macro is exec
 and then the assigned key.  The macro is also executed if you press
 Meta, Ctrl, or Esc and the assigned key, provided that the key is not
 used for any other function.  The macro commands are stored in the file
-.BR ~/.mc/cedit/cooledit.macros .
+.BR ~/.etc/mc/cedit/cooledit.macros .
 Do NOT edit this file if you are going to use macros again in the same
 editing session, because
 .B mcedit
@@ -118,14 +118,14 @@ other running editors for macros to take
 .B F19
 will format C, C++, Java or HTML code when it is highlighted.  An executable
 file called
-.B ~/.mc/cedit/edit.indent.rc
+.B ~/.etc/mc/cedit/edit.indent.rc
 will be created for you from the default template.  Feel free to edit it
 if you need.
 .PP
 .B C-p
 will run ispell on a block of text in a similar way.  The script file
 will be called
-.BR ~/.mc/cedit/edit.spell.rc .
+.BR ~/.etc/mc/cedit/edit.spell.rc .
 .PP
 If some keys don't work, you can use
 .B Learn Keys
@@ -137,12 +137,12 @@ menu.
 supports syntax highlighting.  This means that keywords and contexts
 (like C comments, string constants, etc) are highlighted in different
 colors.  The following section explains the format of the file
-.BR ~/.mc/cedit/Syntax .
+.BR ~/.etc/mc/cedit/Syntax .
 If this file is missing, system-wide
 .B @prefix@/share/mc/syntax/Syntax
 is used.
 The file
-.B ~/.mc/cedit/Syntax
+.B ~/.etc/mc/cedit/Syntax
 is rescanned on opening of a any new editor file.  The file contains
 rules for highlighting, each of which is given on a separate line, and
 define which keywords will be highlighted to what color.
@@ -359,7 +359,7 @@ Most options can now be set from the edi
 the
 .B Options
 menu.  The following options are defined in
-.B ~/.mc/ini
+.B ~/.etc/mc/ini
 and have obvious counterparts in the dialog box.  You can modify them to
 change the editor behavior, by editing the file.  Unless specified, a 1
 sets the option to on, and a 0 sets it to off, as is usual.
@@ -467,24 +467,24 @@ The help file for the program.
 .I @prefix@/share/mc/mc.ini
 .IP
 The default system-wide setup for GNU Midnight Commander, used only if
-the user's own ~/.mc/ini file is missing.
+the user's own ~/.etc/mc/ini file is missing.
 .PP
 .I @prefix@/share/mc/mc.lib
 .IP
 Global settings for the Midnight Commander.  Settings in this file
-affect all users, whether they have ~/.mc/ini or not.
+affect all users, whether they have ~/.etc/mc/ini or not.
 .PP
 .I @prefix@/share/mc/syntax/*
 .IP
 The default system-wide syntax files for mcedit, used only if
-the corresponding user's own ~/.mc/cedit/ file is missing.
+the corresponding user's own ~/.etc/mc/cedit/ file is missing.
 .PP
-.I $HOME/.mc/ini
+.I $HOME/.etc/mc/ini
 .IP
 User's own setup.  If this file is present then the setup is loaded
 from here instead of the system-wide setup file.
 .PP
-.I $HOME/.mc/cedit/
+.I $HOME/.etc/mc/cedit/
 .IP
 User's own directory where block commands are processed and saved and
 user's own syntax files are located.
